internal/cloud/backend_run_warning.go

```diff
@@ -0,0 +1,44 @@
+package cloud
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	tfe "github.com/hashicorp/go-tfe"
+)
+
+const changedPolicyEnforcementAction = "changed_policy_enforcements"
+const changedTaskEnforcementAction = "changed_task_enforcements"
+const ignoredPolicySetAction = "ignored_policy_sets"
+
+func (b *Cloud) renderRunWarnings(ctx context.Context, client *tfe.Client, runId string) error {
+	if b.CLI == nil {
+		return nil
+	}
+
+	result, err := client.RunEvents.List(ctx, runId, nil)
+	if err != nil {
+		return err
+	}
+	if result == nil {
+		return nil
+	}
+
+	// We don't have to worry about paging as the API doesn't support it yet
+	for _, re := range result.Items {
+		switch re.Action {
+		case changedPolicyEnforcementAction, changedTaskEnforcementAction, ignoredPolicySetAction:
+			if re.Description != "" {
```

Comment:
In which cases one of those actions could be set but its description would be empty?

Reply:
The backend API _shouldn't_  emit these events with an empty description but I figured there's no harm in putting in the check and the UX of `Warning: /n` is pretty bad.